lib/generators/geoblacklight/install_generator.rb

```diff
@@ -91,7 +91,17 @@ def update_application_name
       gsub_file('config/locales/blacklight.en.yml', 'Blacklight', 'GeoBlacklight')
     end
 
+    # Ensure that assets/images exists
+    def create_image_assets_directory
+      FileUtils.mkdir_p('app/assets/images') unless File.directory?('app/assets/images')
+    end
+
     def bundle_install
+      # Override the sass-rails problem
+      if Rails.version =~ /5\.[12]/
+        gsub_file('Gemfile', "gem 'sass-rails', '~> 5.0'", "gem 'sass-rails', '~> 6.0'")
```

Comment:
There is a mechanism to resolve this using Gemfile.extra approach. Was that not working?

Reply:
It was not for me, I can make a second attempt.

Reply:
Trying to override the `sass-rails` release with https://github.com/geoblacklight/geoblacklight/pull/876/commits/e586b1485188053cf20a354c1259187d01ed26d5 broke for me locally. Should I instead downgrade some other dependency?

Reply:
Downgrading `sprockets` didn't break anything for me, I can get this rebased with that commit and request another review.

Reply:
Did you `rf -rf .internal_test_app` and `rm Gemfile.lock` and regenerate?

Reply:
I apologize, I spoke too soon. The `Gemfile.extra` approach still seems to be breaking.

Reply:
Do we need to add something like this to the `Gemfile`?

```ruby
if File.exists?('spec/test_app_templates/Gemfile.extra')
  eval File.read('spec/test_app_templates/Gemfile.extra'), nil, 'spec/test_app_templates/Gemfile.extra'
end 
```

The reason I'm advocating for this approach is for consistency with other Blacklight engine projects.

Reply:
Thank you very much, that seems to have fixed everything for me locally.